test/sai_test/config/route_configer.py

```diff
@@ -29,15 +29,15 @@ def t0_route_config_helper(test_obj, is_create_route=True, is_create_route_for_l
         route_configer.create_default_route()
 
     if is_create_route_for_lag:
-        route_configer.create_route_and_neighbor_entry_for_port(ip_addr=test_obj.lag1_ip,
-            mac_addr=test_obj.lag1_nb_mac,
-            port_id=test_obj.lag1.lag_id,
-            virtual_router_id=test_obj.default_vrf)
+        route_configer.create_route_and_neighbor_entry_for_port(ip_addr=test_obj.lag1_neighbor.ipv4,
```

Comment:
please rebase the newest master branch and make change based on that, route and neighbor config for lag has changed.